tests/test_rate_limit.py

```
@@ -1,27 +1,30 @@
 import os
-import pytest
-from unittest.mock import patch, MagicMock
+from unittest.mock import MagicMock, patch
 
 # Set SECRET_KEY for testing if not set
 if "SECRET_KEY" not in os.environ:
-    os.environ["SECRET_KEY"] = "test-secret-key-123"
+    os.environ["SECRET_KEY"] = "test-secret-key-123"  # NOSONAR
```

Ruff (0.15.0)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)